test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -299,6 +300,37 @@ func GetNumaNodes(node *corev1.Node) (map[int][]int, error) {
 	return numaCpus, err
 }
 
+// GetCoreSiblings returns the siblings of core per numa node
+func GetCoreSiblings(node *corev1.Node) (map[int]map[int][]int, error) {
+	lscpuCmd := []string{"lscpu", "-e=node,core,cpu", "-J"}
+	out, err := ExecCommandOnNode(lscpuCmd, node)
+	var result NumaNodes
+	var numaNode, core, cpu int
+	coreSiblings := make(map[int]map[int][]int)
+	err = json.Unmarshal([]byte(out), &result)
+	if err != nil {
+		return nil, err
+	}
+	for _, value := range result.Cpus {
+		if numaNode, err = strconv.Atoi(value.Node); err != nil {
+			break
+		}
+		if core, err = strconv.Atoi(value.Core); err != nil {
```

Comment:
this is redundant right? 
I cannot think in an scenario where returns an error, only if cpu is offline numanode would be -

Reply:
I have addressed this in the latest patch